plugin/trino-hive/src/test/java/io/trino/plugin/hive/s3/TestS3WrongRegionPicked.java

```diff
@@ -36,11 +36,9 @@ public void testS3WrongRegionSelection()
                     .setHiveProperties(ImmutableMap.of("hive.s3.region", "eu-central-1")) // Different than the default one
                     .build()) {
                 String tableName = "s3_region_test_" + randomNameSuffix();
-                queryRunner.execute("CREATE TABLE default." + tableName + " (a int) WITH (external_location = 's3://" + bucketName + "/" + tableName + "')");
-                assertThatThrownBy(() -> queryRunner.execute("SELECT * FROM default." + tableName))
```

Comment:
With this change do we communicate with S3 ?

Reply:
Since we get `getFileStatus` for checking if the path exists. I guess it fails early.